integration-tests/chopsticks/src/tests/xcm/assetHub/__snapshots__/limitedReserveTransferAssetHubSpiritnet.test.ts.snap

```diff
@@ -0,0 +1,194 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
+
+exports[`Limited Reserve Transfers from AH Account Alice -> Spiritnet Account Alice > Spiritnet xcmpQueue 1`] = `
```

Comment:
Can we add the asset that is being transferred? Otherwise it's hard to tell what's going on by just looking at the snapshot, which is actually what snapshots are for.

Reply:
[7170932](https://github.com/KILTprotocol/kilt-node/pull/668/commits/7170932d310e9f1d38f0107b0d53691adebe06ae)

Reply:
`Limited Reserve Transfers from AH Account Alice` is still not specific. What are we transferring? If it's for generic assets other than native, it should be non-native assets.

Reply:
Nvm it's in the end.